From f20d71fb863b6d1f4588b6b175efa9f2fd331d90 Mon Sep 17 00:00:00 2001
From: Frank Denis <github@pureftpd.org>
Date: Mon, 17 Aug 2015 22:41:24 +0200
Subject: [PATCH] Remove network.target dependency from dnscrypt-proxy.socket
 If this is not correct, I'd love some input from omebody who knows systemd
 more than I do (i.e. virtually anyone).

---
 dnscrypt-proxy.socket | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dnscrypt-proxy.socket b/dnscrypt-proxy.socket
index 22e62b9..ea20922 100644
--- a/dnscrypt-proxy.socket
+++ b/dnscrypt-proxy.socket
@@ -1,6 +1,5 @@
 [Unit]
 Description=dnscrypt-proxy listening socket
-After=network.target
 
 [Socket]
 ListenStream=127.0.0.1:53
-- 
2.5.1

